packages/ui-components/src/components/absolute-time-picker-dropdown/absolute-time-picker-dropdown.utils.ts

```diff
@@ -36,7 +36,43 @@ export const getFormattedSelectedDates = (range: SelectedTimestamp, mode: EAbsol
 	return [dateFrom, dateTo];
 };
 
-export const areDatesValidByRange = (range: SelectedTimestamp, mode: EAbsoluteTimePickerMode): boolean => {
+export const getAbsoluteTimePickerError = (range: SelectedTimestamp, mode: EAbsoluteTimePickerMode, limits: IAbsoluteTimeLimits): EAbsoluteTimeError => {
```

Comment:
The function signature also should mark `undefined` as a possible return value

Reply:
Done 👍